reference/configuration/swiftmailer.rst

```diff
@@ -37,6 +41,15 @@ Configuration
 * `disable_delivery`_
 * `logging`_
 
+url
+~~~
+
+**type**: ``string``
+
+The entire SwiftMailer configuration using a DSN-like URL format.
```

Comment:
I think "using" is correct here. We're not saying that Swift "uses" this ... but that the "url" option is all the Swift config using a DSN format.